srml/timestamp/src/lib.rs

```diff
@@ -53,7 +53,7 @@ extern crate parity_codec as codec;
 
 use runtime_support::{StorageValue, Parameter};
 use runtime_support::dispatch::Result;
-use runtime_primitives::traits::{OnFinalise, SimpleArithmetic, Zero};
+use runtime_primitives::traits::{As, OnFinalise, SimpleArithmetic, Zero};
```

Comment:
The `As` trait is already imported some lines below that, but its behind a `#[cfg(feature = "std")]`

Reply:
Thanks!